lib/modules/manager/gradle/extract.ts

```diff
@@ -26,14 +28,33 @@ import {
 
 const datasource = MavenDatasource.id;
 
+function getRegistryUrlsForDep(
+  packageRegistries: PackageRegistry[],
+  dep: PackageDependency<GradleManagerData>
+): string[] {
+  const scope = dep.depType === 'plugin' ? 'plugin' : 'dep';
+
+  const registryUrls = packageRegistries
+    .filter((item) => item.scope === scope)
+    .map((item) => item.registryUrl);
+
+  registryUrls.push(...(dep.registryUrls ?? []));
```

Comment:
isn't an if better here for performance?

Reply:
I've removed the clause now, as it was dead code. `dep.registryUrls` is set only once in `extract.ts`, so this condition would never be true.